apps/sms/test/marionette/lib/views/new-message/view.js

```diff
@@ -74,8 +74,10 @@ NewMessageView.prototype = {
 
   clearRecipients: function() {
     var recipientsList = this.accessors.recipientsList;
-    recipientsList.tap();
+    this.accessors.recipientsInput.tap(); // tapping the available space
```

Comment:
tapping the recipientsList means we tap in the middle of the space; so the bug was that we were taping first in "a@b.com" (in the test itself), then in the middle of the recipientsList space, which was actually tapping somewhere inside "a@b.com" if for some reason there was another recipient before this one.

now we tap right in the available space so that we always tap on the empty space.